firstvoices/backend/models/mtd.py

```diff
@@ -1,11 +1,11 @@
 from django.db import models
 from django.utils.translation import gettext as _
 
-from backend.models.base import BaseSiteContentModel
+from backend.models.jobs import BaseJob
 from backend.permissions import predicates
 
 
-class MTDExportFormat(BaseSiteContentModel):
+class MTDExportFormat(BaseJob):
```

Comment:
Since its inheriting from `BaseJob`, would `MTDExportJob` be a more appropriate name now ?